sbi/utils/pyroutils.py

```diff
@@ -33,3 +40,151 @@ def prior():
             transforms[name] = biject_to(fn.support).inv
 
     return transforms
+
+
+class EstimatorDistribution(pyro.distributions.TorchDistribution):
```

Comment:
given that it has an abstract method below, should this be an `ABC` as well?

Reply:
Good point. It's cleaner to just raise a `NotImplementedError`, so now it does that.